internal/celutils/utils.go

```diff
@@ -0,0 +1,60 @@
+package celutils
+
+import (
+	"github.com/google/cel-go/cel"
+	"github.com/google/cel-go/common/types/ref"
+)
+
+var (
+	celEnv *cel.Env
+)
+
+func init() {
+	env, err := cel.NewEnv(
+		cel.Variable(CELVariableClaims, cel.MapType(cel.StringType, cel.DynType)),
+		cel.Variable(CELVariableSubSHA256, cel.StringType),
+	)
+
+	if err != nil {
+		panic(err)
+	}
+
+	celEnv = env
+}
+
+// ParseCEL parses a CEL expression.
+func ParseCEL(input string) (*cel.Ast, error) {
+	ast, issues := celEnv.Compile(input)
+	if err := issues.Err(); err != nil {
+		wrapped := ErrorCELParse{
```

Comment:
Wanna add `wrap()` for `ErrorCELParse` and `ErrorCELParse` in internal/celutils/utils.go ? I see a couple of repeats.

Reply:
What does a `wrap` function get us that initializing the struct directly doesn't?